pkg/project/project.go

```diff
@@ -18,12 +18,14 @@ import (
 // nolint: gochecknoglobals
 var driveLetterRegex = regexp.MustCompile(`^[a-zA-Z]:\\$`)
 
-// WakaTimeProjectFile is the special file which if present should contain the project name and optional branch name
-// that will be used instead of the auto-detected project and branch names.
-const WakaTimeProjectFile = ".wakatime-project"
+const (
+	// WakaTimeProjectFile is the special file which if present should contain the project name and optional branch name
+	// that will be used instead of the auto-detected project and branch names.
+	WakaTimeProjectFile = ".wakatime-project"
 
```

Comment:
Don't need actually blank space here, let's try to stick to what we already have done for the other files.